lisp/cl-webkit2/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           common_lisp 1.0
+
+github.setup        joachifm cl-webkit 3.5.9
+name                cl-webkit2
```

Comment:
please add `github.tarball_from archive`

Reply:
@reneeotten here it is!

Reply:
It looks like you need to update the checksum, based on the CI failure

Reply:
agr

Reply:
@mascguy , @reneeotten here it is